### PR TITLE
[5.3] Add fluent email priority setter

### PR DESCRIPTION
**Pull request by KKSzymanowski**
Regarding https://github.com/laravel/framework/issues/15153

### CODE REVIEW

**Comment by KKSzymanowski** [comment]
One thing. I haven't found any tests for the `Mailable` class. 
I didn't know where to put tests for this functionality so I didn't write any.

File: src/Illuminate/Mail/Mailable.php [hunk 1]
```diff
@@ -341,6 +341,23 @@ public function replyTo($address, $name = null)
     }
 
     /**
+     * Set the priority of this message.
+     *
+     * The value is an integer where 1 is the highest priority and 5 is the lowest.
```
Comment (GrahamCampbell) [inline src/Illuminate/Mail/Mailable.php]: Are there constants for these inside swiftmailer?

Comment (KKSzymanowski) [inline src/Illuminate/Mail/Mailable.php]: I haven't found any.

This is the SwiftMailer method header:
```php
/**
 * Set the priority of this message.
 *
 * The value is an integer where 1 is the highest priority and 5 is the lowest.
 *
 * @param int $priority
 *
 * @return Swift_Mime_SimpleMessage
 */
public function setPriority($priority);
```

**Comment by KKSzymanowski** [comment]
There are methods `to` and `bcc` on `Illuminate\Mail\Mailer`(so also on `Mail` facade). Should I add `priority` method to `Illuminate\Mail\Mailer` and to `Illuminate\Mail\MailableMailer` classes?

Also, why there's no `cc` method on `Illuminate\Mail\Mailer`?

**Comment by KKSzymanowski** [comment]
@taylorotwell, @GrahamCampbell 
What about `Mail` facade? Should I add the `priority`(or `withPriority`) method there as well or leave it like it is?

**Pull request merged**